profiles/pci/handhelds/profiles.toml

```
@@ -139,8 +134,7 @@ post_remove = """
     done
     echo "Removing audio profile..."
     rm -f /etc/pipewire/pipewire.conf.d/filter-chain.conf
-    rm -f /etc/wireplumber/wireplumber.conf.d/alsa-card0{,-x}.conf
-    rm -f /etc/wireplumber/wireplumber.conf.d/alsa-card1.conf
+    rm -f /etc/wireplumber/wireplumber.conf.d/alsa-card{0,1}.conf
```

Looks like he's no longer needed here either
```suggestion
    rm -f /etc/wireplumber/wireplumber.conf.d/alsa-card0.conf
```

---

It's still used actually, I missed that in the post_install part. Let me fix that.

---

Added it back now.